src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -1265,5 +1265,19 @@ private static int GetDeclarationDepth(SyntaxNode? node)
 
             return base.VisitInterpolatedStringExpression(node);
         }
+
+        public override SyntaxNode? VisitXmlTextAttribute(XmlTextAttributeSyntax node)
+        {
+            var attribute = (XmlTextAttributeSyntax?)base.VisitXmlTextAttribute(node);
+
+            if (attribute is null or { HasTrailingTrivia: true })
+            {
+                return attribute;
+            }
+
+            return GetNextRelevantToken(node.EndQuoteToken).Kind() != SyntaxKind.GreaterThanToken
+                ? attribute.WithTrailingTrivia(GetSpace())
```

Comment:
❗ While this space is generally desirable for highly structured documents (e.g. project files), it causes unnecessary noise and wrapping characteristics for the common self-closing XML elements in documentation (e.g. `<see langword="null"/>` or `<paramref name="text"/>`. There should not be any space between `"` and `/>` for these elements.

Reply:
📝 Note that two other features we have today will generate code without this space:

1. Type `null` and press <kbd>Ctrl</kbd>+<kbd>Space</kbd>
2. Type `null` and press <kbd>Ctrl</kbd>+<kbd>.</kbd> &rarr; **Use &lt;see langword="null"/>**